### PR TITLE
Add setup instruction for assetic bundle

### PR DESCRIPTION
**Pull request by revoltek-daniel**
#### What's in this PR?

Add missing Assetic Bundle install step if installation from min sulu setup project.

#### Why?
Setup themes correctly

### CODE REVIEW

File: book/themes.rst [hunk 1]
```diff
@@ -66,6 +66,36 @@ To enable it add the following lines into the `app/AbstractKernel.php` and
         load_controllers: false
         assetic_integration: true
 
+Make sure that the AsseticBundle is active, if not add
+
+.. code-block:: xml
```
Comment (alexander-schranz) [inline book/themes.rst]: is php

**Comment by alexander-schranz** [comment]
The assetic bundle itself is not needed for theming! You can use the theme bundle without the assetic bundle and just link in the different twig files to other resources.

Its only needed when use the `assetic_integration` or when you include the resources with assetic twig functions.

So in my case `assetic_integration: true` should not be required and the `Assetic` maybe an own section here how to include it. /cc @danrot @wachterjohannes

File: book/themes.rst [hunk 2]
```diff
@@ -66,6 +66,36 @@ To enable it add the following lines into the `app/AbstractKernel.php` and
         load_controllers: false
         assetic_integration: true
 
+Make sure that the AsseticBundle is active, if not add
```
Comment (alexander-schranz) [inline book/themes.rst]: assetic bundle is not required when just using the theme bundle

**Comment by wachterjohannes** [comment]
i would also add an own section - but i am not quite sure if we should handle this in our docu (which has also be maintained here) - when we dont rely on it @danrot what do you think?

**Comment by revoltek-daniel** [comment]
So you should remove the ```assetic_integration: true``` part from the setup guide.

**Comment by danrot** [comment]
What was the actual problem? Assetic should really not be required. Maybe only mention that the `assetic_integration` must be set to `true` when Assetic is used, but I would not explain how to install assetic. For that we can simply refer to the assetic documentation.

**Comment by alexander-schranz** [comment]
assetic_integration has been removed from the docs some time ago to avoid the assitic not installed error.
As it also not recommended to use assetic anymore I will close this PR.

@revoltek-daniel still thank you for your contribution and show as an error in the documentation.

**Pull request closed without merging**